src/target_intacct/statistical_journal.py

```diff
@@ -79,7 +79,7 @@ def load_statistical_journal_entries(
     input_value = get_input()
 
     # Convert input from dictionary to DataFrame
-    data_frame = pd.DataFrame(input_value)
+    data_frame = pd.DataFrame(input_value[0])
```

Comment:
will that always be indexable?

Reply:
based on the function below, it looks like it won't be indexable if there is no input.  Might wish to have some sort of fallback / default value.